### ISSUE TITLE
Support non-root output directory in output repository

### ISSUE THREAD

**Issue by GregBrimble** [GregBrimble/tailwindui-crawler-action#4]

**Comment by entity** [comment]
Yes please!

**Comment by GregBrimble** [comment]
Done! Much easier than I feared, actually :) That's been released as v1.2.0.

Just add the following to your Action inputs:

```diff
# ...
      - name: Run crawler
        uses: gregbrimble/tailwindui-crawler-action@master
        with:
          email: ${{ secrets.TAILWINDUI_EMAIL}}
          password: ${{ secrets.TAILWINDUI_PASSWORD }}
+         outputfolder: ./customoutputfolder
```

And your components will be dumped in there (relative to the root of the repository) instead :)

**Comment by entity** [comment]
> Done! Much easier than I feared, actually :) That's been released as v1.2.0.
> 
> Just add the following to your Action inputs:
> 
> ```diff
> # ...
>       - name: Run crawler
>         uses: gregbrimble/tailwindui-crawler-action@master
>         with:
>           email: ${{ secrets.TAILWINDUI_EMAIL}}
>           password: ${{ secrets.TAILWINDUI_PASSWORD }}
> +         outputfolder: ./customoutputfolder
> ```
> 
> And your components will be dumped in there (relative to the root of the repository) instead :)

Hey, apologies being late on this! Would there be any chance of just automatically reading the .env `OUTPUT` if present? IMO would be cleaner to read, or manual override via the action if desired.

**Comment by GregBrimble** [comment]
Definitely possible, but I've unfortunately not got the time to implement this myself right now. More than happy to accept a PR for this if you're interested, though!
